src/app/security-group/sg-template-list/sg-template-list-item.component.scss

```diff
@@ -16,3 +16,14 @@
 .description span {
   width: auto;
 }
+
+button {
```

Comment:
Please move this code from all the card components to app.scss or create a wrapper component and move styling to that component